changelog.txt

```diff
@@ -1,3 +1,20 @@
+4.38.0
+==========
+### New Features:
+**Disabling cache syncronization for confidential client apps by default** See [2848](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2848)
+**MSAL will now provide the correlation id as part of cache callback (TokenCacheNotificationArgs)** See [3008](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3008)
+**MSAL now provides a key to access the OBO token for long running process** See (2733)[https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2733]
+**Added hybrid SPA support to MSAL** See (2920)[https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2920]
+
+### Bug Fixes:
+==========
+**Fixed issue where the authentication browser pop up would fail to show but no exception is thrown** See [2839](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2839)
```

Comment:
```suggestion
**Fixed issue where the authentication browser pop up would fail to show without an exception being thrown** See [2839](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2839)
```